flink-runtime/src/test/java/org/apache/flink/runtime/highavailability/zookeeper/ZooKeeperHaServicesTest.java

```diff
@@ -224,23 +224,26 @@ private void runCleanupTestWithJob(
             final RunningJobsRegistry runningJobsRegistry =
                     zooKeeperHaServices.getRunningJobsRegistry();
 
-            final LeaderRetrievalUtils.LeaderConnectionInfoListener listener =
+            final LeaderRetrievalUtils.LeaderConnectionInfoListener resourceManagerLeaderListener =
                     new LeaderRetrievalUtils.LeaderConnectionInfoListener();
-            resourceManagerLeaderRetriever.start(listener);
+            resourceManagerLeaderRetriever.start(resourceManagerLeaderListener);
             resourceManagerLeaderElectionService.start(
                     new TestingContender("foobar", resourceManagerLeaderElectionService));
             LeaderElectionService jobManagerLeaderElectionService =
                     zooKeeperHaServices.getJobManagerLeaderElectionService(jobId);
             jobManagerLeaderElectionService.start(
-                    new TestingContender("", jobManagerLeaderElectionService));
+                    new TestingContender("barfoo", jobManagerLeaderElectionService));
             LeaderRetrievalService jobManagerLeaderRetriever =
                     zooKeeperHaServices.getJobManagerLeaderRetriever(jobId);
-            jobManagerLeaderRetriever.start(
-                    new LeaderRetrievalUtils.LeaderConnectionInfoListener());
+            final LeaderRetrievalUtils.LeaderConnectionInfoListener jobMasterLeaderListener =
```

Comment:
nit: `jobManagerLeaderListener` might be a better name for consistency purposes. But I understand that `jobMasterLeaderListener` is the more correct name.

Reply:
Ok, I think this is valid. Will change it.